.travis.yml

```diff
@@ -27,10 +26,9 @@ script:
 - tox
 
 before_deploy:
+- pip3 install bumpversion pypandoc
```

Comment:
we install packages using `pip3` but the cache expects `pip` 🤔  why>?

Reply:
That is just the want that Travis works.  In a Travis CI job that does pip3 installs, do:
* ls -R $HOME/.cache/pip  # -> a bunch of nested directories
* ls -R $HOME/.cache/pip3  # -> ls: cannot access '/home/travis/.cache/pip3': No such file or directory